### ISSUE TITLE
Open source REST API

### ISSUE THREAD

**Issue by JonnyTech** [GabrielTavernini/Covid19Stats#3]
@GabrielTavernini you [mentioned](https://gitlab.com/fdroid/rfp/-/issues/1349#note_345469053) finding an alternative data source for this app. Recently I tested [CoronaTab](https://github.com/PotentialWeb/CoronaTab/wiki/RESTful-API-documentation) for another project. It uses JHU data which made it ideal for me. Maybe it is suitable for your needs. If not then state what is required so something can be found.

**Comment by GabrielTavernini** [comment]
I haven't been able to test the endpoints because right now both the APIs and the website seem to be unreachable. Despite that, I was able to take a look at the documentation and it looks pretty solid but some data is missing (like testing data, per Mln data, etc...).

**What's required:**
I'm looking for a simple and open API that has all the data that's currently being used in the app.

**Comment by JonnyTech** [comment]
Maybe you missed this:

>We have Origin CORS policies set to only allow coronatab.app domain, so you won't be able to access our API directly from any site, therefore we encourage you to fetch the data and store it on your server, or another option is to setup a Reverse proxy.

Use the [Corona Data Scraper](https://coronadatascraper.com) to pull the data (ie [CSV](https://coronadatascraper.com/data.csv) or [JSON](https://coronadatascraper.com/data.json)), you will see that for each country it lists population so it is trivial to calculate 'per million' values. There is also 'tested' data - although not all countries report this.

Anyway, I shall take another look later.

**Comment by GabrielTavernini** [comment]
I did see that but at the moment even the website [coronatab.app](https://coronatab.app/) appears to be unreachable.

I took a brief look at one of [Corona Data Scraper](https://coronadatascraper.com/)'s JSON files and it appears to be exactly what the app needs. Unfortunately, I don't have much time right now due to school exams and college applications but as soon as I find some time I'll try to modify the app to use [CDS](https://coronadatascraper.com/) as its data source.
